ddtrace/tracer.py

```diff
@@ -34,12 +34,17 @@ def __init__(self):
         Create a new ``Tracer`` instance. A global tracer is already initialized
         for common usage, so there is no need to initialize your own ``Tracer``.
         """
+        self.sampler = None
+        self.distributed_sampler = None
+
         # Apply the default configuration
         self.configure(
             enabled=True,
             hostname=self.DEFAULT_HOSTNAME,
             port=self.DEFAULT_PORT,
             sampler=AllSampler(),
+            # TODO: by default, a ServiceSampler periodically updated
+            distributed_sampler=RateByServiceSampler(),
```

Comment:
In fact, as a default we will want the `distributed_sampler` to be `None` so that we don't use the priority logic / we keep using the signature sampling (which is much more mature / interesting with non-distributed traces ).